packages/tasks/src/tasks/video-text-to-text/data.ts

```diff
@@ -0,0 +1,59 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "Multiple-choice questions and answers about videos.",
+			id: "lmms-lab/Video-MME",
+		},
+        {
+			description: "A dataset of instructions and question-answer pairs about videos.",
+			id: "lmms-lab/VideoChatGPT",
+		},
+		
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "video-text-to-text-input.gif",
+				type: "video",
+			},
+			{
+				label: "Text Prompt",
+				content: "What is happening in this video?",
+				type: "text",
+			},
+		],
+		outputs: [
+			{
+				label: "Answer",
+				content:
+					"The video shows a series of images showing a fountain with water jets and a variety of colorful flowers and butterflies in the background.",
+				type: "text",
+			},
+		],
+	},
+	metrics: [],
+	models: [
+		{
+			description: "A robust video-text-to-text model that can take in image and video inputs.",
+			id: "llava-hf/llava-interleave-qwen-7b-hf",
```

Comment:
can we change this to llava-onevision, that is better than interleave and I'll merge the PR for it soon